CHANGES.md

```diff
@@ -30,7 +30,13 @@ OpenSSL 3.4
 
 ### Changes between 3.4 and 3.5 [xx XXX xxxx]
 
- * none yet
+ * Optionally allow the FIPS provider to use the `JITTER` entropy source.
+   Note that using this option will require the resulting FIPS provider
+   to undergo entropy source validation by the [CMVP], without this
+   the FIPS provider will not be FIPS compliant.  Enable this using the
+   configuration option `enable-fips-jitter`.
+
```

Comment:
ESV certification is separate from CMVP certification. When submitting for CMVP certification, one should already obtain ESV certificate. The same ESV certificate can be used multiple times for multiple CMVP submissions. The way the jitter entropy APIs are used, and the way it is linked statically, it is possible to validate jitter entropy library as a static library, and reuse it for multiple submissions. See for example public use document of https://csrc.nist.gov/projects/cryptographic-module-validation-program/entropy-validations/certificate/191

So this caveat is more strict than it actually is.

The idea is that many/most vendors have jitter entropy certified, and with these changes they can reuse their builds straight away. As most vendors have ESV for jitterentropy in kernel/openssl/gcrypt today, and this change makes it possible to reuse ESV, rather than require resubmissions each time. Previously chaining was causing the ESV to be non-reusable, and needed to reobtain ESV for every OpenSSL build change, due to OpenSSL DRBG being in the ESV boundary; which it no-longer is thanks to static linking of the standalone build/library and unchaining.